x-pack/plugins/secops/public/components/notes/helpers.tsx

```diff
@@ -100,7 +71,7 @@ export const createNote = ({
   id: getNewNoteId(),
   lastEdit: null,
   note: newNote.trim(),
-  user: 'admin', // TODO: get the logged-in Kibana user
+  user: 'elastic', // TODO: get the logged-in Kibana user
```

Comment:
Do we have a ticket written up for this for our backlog for tracking? If not, I would add one and add the link here to keep track of our work.

Reply:
There is an issue created on Dec 17 for this: https://github.com/elastic/ingest-dev/issues/152